api/who/who.proto

```diff
@@ -4,11 +4,23 @@ package who;
 
 message Void {}
 
-// Example service
-service EchoService {
-  rpc echo(EchoMessage) returns (EchoMessage);
+service WhoService {
+
+  rpc putDeviceToken(PutDeviceTokenRequest) returns (Void);
+  rpc putLocation(PutLocationRequest) returns (Void);
+}
+
+message PutDeviceTokenRequest {
+  string token = 1;
+}
+
+message PutLocationRequest {
```

Comment:
We're adding random jitter to this on the client-side right?  Maybe a comment right here to avoid any confusion.

Reply:
Correct, if we're adding noise to the location, it's best to do so in the client.